app/controllers/registration_wizard_controller.rb

```diff
@@ -2,7 +2,8 @@ class RegistrationWizardController < ApplicationController
   def show
     @wizard = RegistrationWizard.new(current_step: params[:step].underscore,
                                      store: store,
-                                     request: request)
+                                     request: request,
+                                     params: wizard_params)
```

Comment:
This is to temporarily allow the versions param to be passed through into the forms via a GET param. That way we can override the genuine page that would otherwise be rendered. This will be removed before launch, it is a temporary measure for copy testing.